src/docfx/build/page/ContributionInfo.cs

```diff
@@ -14,6 +14,7 @@ public class ContributionInfo
     {
         public string UpdateAt { get; set; }
 
+        [JsonIgnore]
```

Comment:
`JsonIgnore` will affect during de-serialization as well, is that what we want here?